src/compiler/program.ts

```diff
@@ -70,7 +70,7 @@ namespace ts {
     }
 
     function loadNodeModuleFromFile(candidate: string, failedLookupLocation: string[], host: ModuleResolutionHost): string {
-        return forEach(moduleFileExtensions, tryLoad);
+        return forEach(supportedJsExtensions, tryLoad);

```

Comment:
Is this a safe change? I don't exactly know what's going on here, but it means that we'll look for .js files as well as .ts files even in the pure TypeScript case.